app/routes/questionnaire.py

```diff
@@ -310,9 +311,10 @@ def relationships(
 
 @post_submission_blueprint.route("thank-you/", methods=["GET", "POST"])
 @with_session_store
+@with_questionnaire_store
 @with_schema
-def get_thank_you(schema, session_store):
-    thank_you = ThankYou(schema, session_store)
+def get_thank_you(schema, questionnaire_store, session_store):
+    thank_you = ThankYou(schema, session_store, questionnaire_store.submitted_at)
```

Comment:
Would be more readable if the order of the arguments for the route matched Thank You so:
```
@with_questionnaire_store
@with_session_store
@with_schema
def get_thank_you(schema, session_store, questionnaire_store):
    thank_you = ThankYou(schema, session_store, questionnaire_store.submitted_at)
```